docs/docs/forms.md

```diff
@@ -51,8 +53,10 @@ class NameForm extends React.Component {
   render() {
     return (
       <form onSubmit={this.handleSubmit}>
-        Name:
-        <input type="text" value={this.state.value} onChange={this.handleChange} />
+        <label>
+          Name:
+          <input type="text" value={this.state.value} onChange={this.handleChange} />
+        </label>
```

Comment:
Small request but it's important. We use a non-standard markdown syntax for the opening of code fences. You can see it at the top of this block. That's used to highlight specific lines when rendered on the website. In this case you're adding a line before the last highlight, we w'll end up highlighting the wrong line (I think just need to change this one 23 -> 24, the other ones will likely need a change too, only if the lines changed in the block are above or within the ranges). You can see this in action: https://facebook.github.io/react/docs/forms.html#controlled-components

Sorry it's not obvious. We've broken these a number of times and the only way you'd know was if you loaded the rendered website locally, which most people don't do.

If you could update with that, then this will be good to go. Thanks!